### PR TITLE
Add few more Android clients to the list

### PR DESCRIPTION
**Pull request by shuvashish76**
- URLCheck ([Custom patterns](https://github.com/TrianguloY/UrlChecker/wiki/Custom-patterns))
- LinkSheet (Supports _Experimental_: LibRedirect integration, see [here](https://github.com/1fexd/libredirectkt))

These 2 are the active ones from the [list](https://github.com/offa/android-foss#-url-manipulation).
- Clipious (Invidious client)
- TranslateYou ([Multiple engines](https://github.com/you-apps/TranslateYou#supported-translation-engines) support)

### CODE REVIEW

File: faq.html [hunk 1]
```diff
@@ -182,18 +182,24 @@ <h3 id="chrome_web_store">
...
+        URL Manipulation <span>&#8594;</span>
+        <a href="https://github.com/TrianguloY/UrlChecker">URLCheck</a>,
+        <a href="https://github.com/1fexd/LinkSheet">LinkSheet</a>
+      </li>
       <li>
         YouTube <span>&#8594;</span>
-        <a href="https://newpipe.net/">NewPipe</a>,
-        <a href="https://libretube.dev/">LibreTube</a>
+        <a href="https://libretube.dev/">LibreTube</a>,
+        <a href="https://github.com/lamarios/clipious"></a>,
```
Comment (ManeraKai) [inline faq.html]: Add the name of Clipious here.

File: faq.html [hunk 2]
```diff
@@ -182,18 +182,24 @@ <h3 id="chrome_web_store">
...
+        <a href="https://newpipe.net/">NewPipe</a>
       </li>
       <li>
         Twitter <span>&#8594;</span>
         <a href="https://fritter.cc/">Fritter</a>
       </li>
       <li>
         Translate <span>&#8594;</span>
-        <a href="https://manerakai.github.io/simplytranslate_mobile/">SimplyTranslate Mobile</a>
```
Comment (ManeraKai) [inline faq.html:196]: No need to remove SimplyTranslate Mobile. It's mine bruh.

Comment (shuvashish76) [reply faq.html:196]: Oopsie 🙈, OK 👍🏿

**Comment by ManeraKai** [comment]
Thanks :)

**Pull request merged**